api/tests/functional-tests/backend/core/test_label.py

```diff
@@ -281,6 +282,34 @@ def test_get_labels(
     }
 
 
+def test_get_paginated_labels(
+    db: Session,
+    create_dataset_model,
+):
+    assert len(db.query(models.Label).all()) == 5
+    labels, headers = get_paginated_labels(db)
+    assert len(labels) == 5
+    assert set(labels) == {
+        schemas.Label(key="k1", value="v1"),
+        schemas.Label(key="k2", value="v3"),
+        schemas.Label(key="k1", value="v2"),
+        schemas.Label(key="k1", value="v3"),
+        schemas.Label(key="k3", value="v3"),
+    }
+    assert headers == {"content-range": "items 0-4/5"}
+
+    pred_labels, headers = get_paginated_labels(
+        db, ignore_groundtruths=True, offset=1, limit=100
+    )
+    assert len(pred_labels) == 2
```

Comment:
Can you add a value check to show that the output is deterministic?

Reply:
added!

Reply:
this test isn't deterministic, unfortunately, because the `created_at` times vary across runs, and are often the same for multiple items. this means the sort order will vary across runs, and hence the output here will vary across runs. here's an example of `label` from one run:

<img width="592" alt="image" src="https://github.com/Striveworks/valor/assets/26579930/0540df16-9e15-4f9e-9d63-86d16f4f8846">

Reply:
we could sort by key / value in addition to created_at if we wanted to, but doing so would introduce a small performance penalty for little added value IMO

Reply:
The value itself isn't what needs to be tested, I guess what im trying to say is the following:

```
labels_set == 
get_paginated_labels(
        db, ignore_groundtruths=True, offset=1, limit=100
    ) + get_paginated_labels(
        db, ignore_groundtruths=True, offset=0, limit=1
    )
```

to show that the queries are consistent.

Reply:
This type of test should exist for each implementation of pagination so that you can be certain that arbitrary offsets and limits will recombine to the same set.

Reply:
good idea!